ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
 Similar to the Maven plug-in, the link:https://github.com/OpenLiberty/ci.gradle[Open Liberty Gradle plug-in] provides tasks for you to manage your server and applications.
-
-After you install the Gradle plug-in, include the following lines of code at the top of your `build.gradle` file to specify required plug-ins:
-----
-apply plugin: 'war'
-apply plugin: 'liberty'
-----
-Then, run the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyPackage.md#libertypackage-task[`libertyPackage` task] to package your application and Open Liberty into a WAR file.
-Once you create this server package, you can containerize and deploy it.
 To learn more about the Open Liberty Gradle plug-in, check out this guide on link:https://openliberty.io/guides/gradle-intro.html[Building a web application with Gradle].
 
-=== Runnable JAR file
-You can also use Maven or Gradle to package your application with Open Liberty into a runnable JAR file that contains the application and server runtime.
+== Server packages
+Open Liberty supports different server package types.
+The supported package types are `zip`, `jar`, `tar`, and `tar.gz`.
+If you don't specify a package type, then `zip` is the default, meaning that your application and an Open Liberty installation are packaged into a ZIP file.
+
+For more information about Open Liberty packaging options with Maven, see the link:https://github.com/OpenLiberty/ci.maven/blob/master/docs/package.md#package[`package` goal in the Maven documentation]. For more information about Open Liberty packaging options with Gradle, see the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyPackage.md#libertypackage-task[`libertyPackage` task in the Gradle documentation].
+
+=== Packaging for containerization
+Use Maven or Gradle to package an application before you containerize it.
+Maven and Gradle both work well as part of continuous integration and continuous deployment (CI/CD) pipelines.
+
+When you package an application for containerization and cloud deployment, specify `usr` in the `include` parameter or attribute.
+This option packages only relevant files in the `wlp.user.dir` directory, effectively excluding the runtime binary files from the compressed file.
+
+=== Packaging into a WAR deployment artifact
+You can also choose to package your application into a WAR deployment artifact for deployment on an Open Liberty server.
+Use Maven or Gradle to package your application into a WAR file so that it can be manually deployed on Open Liberty.
+
+Specify `war` as the packaging type in your `pom.xml` file (for Maven) or in your `build.gradle` file (for Gradle). Then, package your application with Maven or Gradle. The result is a WAR deployment artifact that you can deploy on Open Liberty.
+
+=== Packaging into a runnable JAR file
+If you want to package locally, you can package your application and an Open Liberty installation into a link:/docs/ref/general/#runnablejarfiles.html[runnable JAR file].
```

Comment:
I'm not sure I understand the intent of the "if you want to package locally". How is this different from the other options?